code/datums/brain_damage/split_personality.dm

```diff
@@ -238,5 +239,36 @@
 	if(objective)
 		to_chat(src, span_notice("Your master left you an objective: <b>[objective]</b>. Follow it at all costs when in control."))
 
+/datum/brain_trauma/severe/split_personality/blackout
+	name = "Alcohol-Induced CNS Impairment"
+	desc = "Patient's CNS has been temporarily impaired by imbibed alcohol, blocking memory formation, and causing reduced cognition and stupefaction."
+	scan_desc = "alcohol-induced CNS impairment"
+	gain_text = span_warning("Crap, that was one drink too many. You black out...")
+	lose_text = "You wake up very, very confused and hungover. All you can remember is drinking a lot of alcohol... what happened?"
+	poll_role = "blacked out drunkard"
+	var/duration = 300 //Should last 5 minute
```

Comment:
1. Autodoc
2. This should indicate it's tracking seconds and not deciseconds (`duration_in_seconds`)